<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseWidget</class>
 <widget class="QWidget" name="DatabaseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QListView" name="DTCList"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item>
            <widget class="QLineEdit" name="DTCSearchBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="DTCSearchButton">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QComboBox" name="tableSelection"/>
         </item>
         <item>
          <widget class="QComboBox" name="columnSelection"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="addDTCButton">
         <property name="text">
          <string>Add New DTC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeDTCButton">
         <property name="text">
          <string>Remove Selected DTC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editDTCButton">
         <property name="text">
          <string>Edit Selected DTC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="configureDatabaseButton">
         <property name="text">
          <string>Database Configuration</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
